### ISSUE TITLE
Missing Method Exception: Microsoft.EntityFrameworkCore.Metadata.IEntityType.GetProperties()

### ISSUE THREAD

**Issue by overlego** [thepirat000/Audit.NET#632]
**Describe the bug**
Currently trying .net 8 preview and there's an error in saving the changes.

MissingMethodException: Method not found: 'System.Collections.Generic.IEnumerable`1<Microsoft.EntityFrameworkCore.Metadata.IProperty> Microsoft.EntityFrameworkCore.Metadata.IEntityType.GetProperties()'.
Audit.EntityFramework.DbContextHelper.GetChanges(IAuditDbContext context, EntityEntry entry)

**To Reproduce**
Overriding DbContext SaveChanges
```
public class MyContext : DbContext
{
    private readonly DbContextHelper _helper = new DbContextHelper();
    private readonly IAuditDbContext _auditContext;

    public MyContext(DbContextOptions<MyContext> options) : base(options)
    {
        _auditContext = new DefaultAuditContext(this);
        _helper.SetConfig(_auditContext);
    }

    public override int SaveChanges(bool acceptAllChangesOnSuccess)
    {
        return _helper.SaveChanges(_auditContext, () => base.SaveChanges(acceptAllChangesOnSuccess));
    }

    public override async Task<int> SaveChangesAsync(bool acceptAllChangesOnSuccess, CancellationToken cancellationToken = default(CancellationToken))
    {
        return await _helper.SaveChangesAsync(_auditContext, () => base.SaveChangesAsync(acceptAllChangesOnSuccess, cancellationToken));
    }
}
```

**Expected behavior**
Error should not happen and audit changes should be saved to logs.

**Libraries (specify the Audit.NET extensions being used including version):**
 - Audit.NET 21.1.0
 - Audit.EntityFramework.Core 21.1.0
 - Audit.NET.Serilog 21.1.0
 - Microsoft.EntityFrameworkCore: 8.0.0-rc.2.23480.1

**Target .NET framework:**
 - .NET 8

**Additional context**
Using serilog extension to log the audit to a file.
Trying out .net 8 to prepare for the release and see possible breaking changes

**Comment by thepirat000** [comment]
Audit.EntityFramework typically faces compatibility issues with each significant EF Core update, primarily due to its reliance on undisclosed internal components and the change tracker representation, both of which tend to undergo modifications in major version releases. Consequently, I generally do not give support for pre-release versions of EF Core

**Comment by thepirat000** [comment]
This error was fixed in version 22.0.0 targeting NET 8. 

Please upgrade your references and re-test.

Feel free to open a new issue if you find any other incompatibility.

**Comment by overlego** [comment]
@thepirat000 thank you very much! This is now working on the latest version.
